.zenodo.json

```diff
@@ -43,6 +43,12 @@
       "name": "Esteban, Oscar",
       "orcid": "0000-0001-8435-6191"
     }
+    {
```

Comment:
missing comma above - please add your name on top of the list :)

Reply:
Good catch, thanks!